core/outbound/dispatcher_test.go

```diff
@@ -89,7 +91,7 @@ func testCache(t *testing.T) {
 	exchange(questionDomain, dns.TypeA)
 	now := time.Now()
 	exchange(questionDomain, dns.TypeA)
-	if time.Since(now) > 10 * time.Millisecond {
+	if time.Since(now) > 10*time.Millisecond {
```

Comment:
dosen't have to delete the space

Reply:
Auto refomatted by IDE, I'll correct it

Reply:
I'm wonder if we should revert this change. This change is automatically performed by go fmt